release-notes/3.0/preview/api-diff/preview4/standalone-packages/3.0-preview4-standalone-packages_System.Runtime.CompilerServices.md

```diff
@@ -0,0 +1,81 @@
+# System.Runtime.CompilerServices
+
+``` diff
+-namespace System.Runtime.CompilerServices {
```

Comment:
I don't think this diff is correct. We didn't remove these APIs (they are still there within the NuGet package): https://www.nuget.org/packages/System.Runtime.CompilerServices.Unsafe/4.6.0-preview4.19212.13